avm/res/app/session-pool/tests/e2e/max/dependencies.bicep

```diff
@@ -0,0 +1,19 @@
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Managed Identity to be created.')
+param managedIdentityName string
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2023-07-31-preview' = {
+  name: managedIdentityName
+  location: location
+}
+
```

Comment:
Suggest to only keep outputs referenced in the main.test.bicep and remove the rest. For example in this case only `managedIdentityResourceId` is used in the main test file, so the other outputs are not needed